README.md

```diff
@@ -60,39 +73,165 @@ make prove_sp1_ecdsa
 **Blockchain state diff**:
 
 ```bash
-make prove_risc0_blockchain_state
+make prove_risc0_json
+```
+
+```bash
+make prove_sp1_json
+```
+
+**Blockchain state diff**:
+
+```bash
+make prove_risc0_json
+```
+
+```bash
+make prove_sp1_json
+```
+
+**Regex**:
+
+```bash
+make prove_risc0_regex
+```
+
+```bash
+make prove_sp1_regex
+```
+
+**Sha**:
+
+```bash
+make prove_risc0_sha
+```
+
+```bash
+make prove_sp1_sha
+```
+
+**Tendermint**:
+
+```bash
+make prove_risc0_tendermint
+```
+
+```bash
+make prove_sp1_tendermint
+```
+
+**Zk Quiz**:
+
+```bash
+make prove_risc0_zkquiz
 ```
 
 ```bash
-make prove_sp1_blockchain_state
+make prove_sp1_zkquiz
 ```
 
 ## Usage:
 
-To use zkRust, define the code you would like to generate a proof for in a `main.rs` in a directory with the following structure:
+To use zkRust, To use zkRust users specify a `fn main()` whose execution is proven within the zkVM. This function must be defined within a `main.rs` file in a directory with the following structure:
 
 ```
 .
 └── <PROGRAM_DIRECTORY>
     ├── Cargo.toml
     └── src
         └── main.rs
+```
+
+For using more complex programs you can import a separate lib/ crate into the .
```

Comment:
I guess this line needs some changes. Maybe 'lib/crate' should be joined and the end of the line should be fixed.